doc/api/child_process.md

```diff
@@ -110,6 +111,13 @@ exec('my.bat', (err, stdout, stderr) => {
   }
   console.log(stdout);
 });
+
+// Script with spaces in the filename:
+const bat = spawn('"my script.cmd"', ['a', 'b'], { shell:true });
+// or:
+exec('"my script.cmd" a b', (err, stdout, stderr) => {
```

Comment:
No, they won't. Final Winapi call will have the same parameter (something like `cmd.exe /s /k ""my script.cmd" a b"`)